HackingWithSwift/week1/day_4/Skipping items/Skipping items .playground/Contents.swift

```diff
@@ -0,0 +1,8 @@
+for a in 1...10 {
+    
+    if a .isMultiple(of: 2) {
```

Comment:
Remove empty space after the `a`. Besides that try to use meaningful names. In this case you could use `number` instead of `a`.

Reply:
In addition, the formatting here is also a bit of a mess. Take a look :)